app/src/main/res/layout/activity_main.xml

```diff
@@ -13,10 +14,19 @@
         android:layout_height="wrap_content"
         android:layout_gravity="top|center" />
 
-    <android.support.v7.widget.RecyclerView
+    <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recycleView"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"></androidx.recyclerview.widget.RecyclerView>
```

Comment:
you can use "/>" instead of "></androidx.recyclerview.widget.RecyclerView>"

Reply:
done